tuf-spec.md

```diff
@@ -399,7 +399,7 @@ publicly available in the [TAP directory](https://github.com/theupdateframework/
 
 An application uses the framework to interact with one or more repositories.
 A repository is a conceptual source of target files of interest to the
-application.  Each repository has one or more mirrors which are the actual
+application.  Each repository MAY have one or more mirrors which are the actual
```

Comment:
The phrase is a bit ambiguous. It could be read in way that target files are always only provided via mirrors, which is a bit confusing if mirrors are optional. Would it be clearer to say e.g.
```suggestion
application.  Each repository MAY have one or more mirrors as the
```
?

Reply:
This phrasing works for me, thanks @lukpueh